pkgs/applications/misc/ulauncher/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchurl, python27Packages,
+  gnome3, gobjectIntrospection, wrapGAppsHook, gtk3, webkitgtk,
+  libnotify, keybinder3, libappindicator, intltool, wmctrl }:
+
+python27Packages.buildPythonApplication rec  {
+  pname = "ulauncher";
+
+  versionNumber = "4.3.2";
```

Comment:
No need to have this separately.

Reply:
I like it when there's a suffix for the version that's not standard in nixpkgs to do

```nix
let

  versionSuffix = "r8";

in

...
  version = "4.3.2";

  url = ...ulauncher_${version}.${versionSuffix}
```
But I don't think nixpkgs-update would like that, but I still think it's cleaner.

Reply:
Is it not part of version? I do not think repology cares: https://repology.org/project/ulauncher/versions

Reply:
Oops, thought that the manual had a restriction for versions not having `A-Z` in them.
https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Note that the number after `r` distinguishes different sequential releases.
https://github.com/Ulauncher/Ulauncher/releases

Since the manual uses an example with a-z characters (`0.3.1rc2`), should we still ignore the suffix of this package's versions?

Reply:
> Note that the number after r distinguishes different sequential releases.
https://github.com/Ulauncher/Ulauncher/releases

That is an interesting choice, I think we should keep it because of that.
It's an important part of the identifier.

Reply:
That is, `version` should be the value of that tag.

Reply:
Turns out that `versionNumber` isn't used anymore. I pushed a commit to simply use the version variable and not do anything else weird https://github.com/NixOS/nixpkgs/pull/57859/commits/38bf10adc6d2022741d2d77d35396bb06962d39f